specification/voiceservices/data-plane/Microsoft.VoiceServices/stable/2023-10-01/swagger.json

```
@@ -0,0 +1,1015 @@
+            "minLength": 1,
+            "maxLength": 100,
+            "required": true,
+            "type": "string",
+            "description": "Name of the account. Can only contain letters, numbers, underscores and dashes. Can be up to 100 characters in length."
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "An error response object.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/errorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete an account": {
+            "$ref": "./examples/DeleteAccount.json"
+          }
+        },
+        "tags": [
+          "Account"
+        ],
+        "description": "Deletes an account (representing a customer)."
+      },
+      "get": {
+        "operationId": "Account_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "accountName",
+            "pattern": "^([A-Za-z0-9_-]+)$",
+            "minLength": 1,
+            "maxLength": 100,
+            "required": true,
+            "type": "string",
+            "description": "Name of the account. Can only contain letters, numbers, underscores and dashes. Can be up to 100 characters in length."
+          },
+          {
+            "name": "status",
+            "in": "query",
+            "default": false,
+            "required": false,
+            "type": "boolean",
+            "description": "Additionally return the provisioning state of the backends configured for this account (e.g. DNS subdomain provisioning for Microsoft Teams Direct Routing)."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AccountWithDnsStateResponseDto"
+            }
+          },
+          "default": {
+            "description": "An error response object.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/errorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get information about an account": {
+            "$ref": "./examples/GetAccount.json"
+          }
+        },
+        "tags": [
+          "Account"
+        ],
+        "description": "Gets the details of the specified account. An account represents a customer."
+      },
+      "put": {
+        "operationId": "Account_Put",
```

The operationId should be descriptive of the operation and not the underlying HTTP method used.
```suggestion
        "operationId": "Account_CreateOrReplace",
```

---

Updated. Checked other operationIds too.